docs/sql-ref-syntax-ddl-alter-table.md

```diff
@@ -83,6 +83,37 @@ ALTER TABLE table_identifier ADD COLUMNS ( col_spec [ , col_spec ... ] )
 </dl>
 
 
+### ADD PARTITION
+
+`ALTER TABLE ADD PARTITION` adds a new partition to an existing, partitioned table.
+
+#### Syntax
+
+```sql
```

Comment:
I don't know why we are using Jinja directives when basic Markdown suffices, but if we really want to stick to the more verbose `{% highlight sql %}` I am happy to switch to that.